### PR TITLE
Stop converting the filename to lowercase

### LINKED ISSUE #179: path to file is lowercased
When I enter full path to file in output the path is lowecased. This leads to errors in for example atom-pydocstyle because it opens files which don't exists.
```
[fdobrovolny@minos sis]$ pydocstyle /home/fdobrovolny/DEV/sis/.../function_based.py
/home/fdobrovolny/dev/sis/.../function_based.py:12 in public function `paginate`:
        D400: First line should end with a period (not 's')
```
(...=path is shortened)
```
[fdobrovolny@minos sis]$ pydocstyle --version
1.0.0
```

### PR DESCRIPTION
**Pull request by sigmavirus24**
This leads to problems for tools integrating with pydocstyle, e.g., atom
linter-pydocstyle

Closes #179 

cc @BrnoPCmaniak

### CODE REVIEW

**Comment by Nurdok** [comment]
Notice that the tests fail for the PR - see the Travis build.
Also, this should be checked on a Windows machine - I think the `lower` call was made because there where "c:\" - "C:\" incosistencies. I'll have to check this later.

**Comment by sigmavirus24** [comment]
> Notice that the tests fail for the PR - see the Travis build.

I did notice that and I was attempting to determine why before I had to leave yesterday. :)

> Also, this should be checked on a Windows machine 

Should we attempt to set up AppVeyor for pydocstyle?

For whatever it may be worth, pycodestyle doesn't munge the paths for case sensitivity and doesn't appear to have problems on Windows.

**Comment by sigmavirus24** [comment]
So I updated the test case that was failing. I also updated the tests for a few things I noticed (that I'll happily pull into a separate pull request) including parametrizing the test that was failing and fixing a warning that pytest wasn't printing by default (and printing pytest warings by default).

File: src/tests/test_integration.py [hunk 1]
```diff
@@ -21,7 +21,7 @@
 __all__ = ()
 
 
-class TestEnv(object):
+class FakeTestEnv(object):
```
Comment (Nurdok) [inline src/tests/test_integration.py]: Why this name change?

Comment (sigmavirus24) [inline src/tests/test_integration.py]: So I'm happy to pull d99705e into a separate pull request. I just noticed it and fixed it while working on this.

Comment (Nurdok) [inline src/tests/test_integration.py]: Oh, this is because `pytest` tries to collect this class? Then it's fine to change the name. However, I don't like "fake" because is creates a real file system environment. How about `InvocationEnv` / `StagingEnv` / `DeployEnv`, etc.?

Comment (The-Compiler) [inline src/tests/test_integration.py]: Yup - pytest tries to collect classes named `Test*`

**Comment by Nurdok** [comment]
I merged this manually and changed `FakeTestEnv` to `SandboxEnv`. Thanks!

**Comment by sigmavirus24** [comment]
Sorry about that. I was going to get to it tonight. :)
